src/labels/CustomLabels.labels

```diff
@@ -8663,6 +8671,30 @@ To check your existing settings, go to Duplicate Rules under Setup. For any rule
         <shortDescription>stgHelpUseDatedConvRates</shortDescription>
         <value>When checked, Dated Exchange Rates are used for currency conversion. If unchecked, the standard exchange rate is used even if Advanced Currency Management is enabled.</value>
     </labels>
+    <labels>
+        <fullName>stgInstallmentOppDisableAll</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>stgInstallmentOppDisableAll</shortDescription>
+        <value>Disable All Installments</value>
+    </labels>
+    <labels>
+        <fullName>stgInstallmentOppDisableFirst</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>stgInstallmentOppDisableFirst</shortDescription>
+        <value>Disable First Installment</value>
+    </labels>
+    <labels>
+        <fullName>stgInstallmentOppEnableAll</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>stgInstallmentOppEnableAll</shortDescription>
+        <value>Always Create Next Installment</value>
```

Comment:
Will this be the default setting?

Reply:
Yes, 'Always Create Next Installment' will be the default